service/controller/cluster/handler/clusterownerreference/controlplane.go

```diff
@@ -46,7 +45,7 @@ func (r *Resource) ensureControlPlane(ctx context.Context, cluster *capi.Cluster
 }
 
 func (r *Resource) updateControlPlaneObject(ctx context.Context, cluster *capi.Cluster, azureMachine *capz.AzureMachine) error {
-	r.logger.Debugf(ctx, "message", fmt.Sprintf("ensuring %s label and 'ownerReference' fields on AzureMachine '%s/%s'", capi.ClusterLabelName, azureMachine.Namespace, azureMachine.Name))
+	r.logger.Debugf(ctx, "message", "ensuring %s label and 'ownerReference' fields on AzureMachine '%s/%s'", capi.ClusterLabelName, azureMachine.Namespace, azureMachine.Name)
```

Comment:
The `"message", ` part should be snapped from these. The first string after `context.Context` is the `message` :slightly_smiling_face: